contrib/gotify/template.py

```diff
@@ -0,0 +1,36 @@
+pkgname = "gotify"
+pkgver = "2.5.0"
+pkgrel = 0
+build_style = "go"
+make_build_args = ["-ldflags", f"-X main.Version={pkgver} -X main.Mode=prod"]
+hostmakedepends = ["go", "yarn"]
+pkgdesc = "Server for sending and receiving real-time messages"
+maintainer = "ttyyls <contact@behri.org>"
+license = "MIT AND CC-BY-4.0"
+url = "https://gotify.net"
+source = f"https://github.com/gotify/server/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "209eb48ac233ff8d9d67b405c4063cb5a33612688e1e8183ebade2128c69c798"
+tool_flags = {"CFLAGS": ["-D_LARGEFILE64_SOURCE"]}
```

Comment:
```suggestion
go_build_tags = ["libsqlite3"] 
```

Reply:
you don't need these cflags

Reply:
some tests fail otherwise

Reply:
because you also need `make_check_args = ["-tags", "libsqlite3", "./..."]`

for a moment i thought they 'failed otherwise' for some different reason since you didn't write why they failed, but it's just the same thing

Reply:
ah nvm

just go_check_tags